client/devdocs/design/index.jsx

```diff
@@ -125,3 +137,33 @@ export default React.createClass( {
 		);
 	}
 } );
+
+DesignAssets.propTypes = {
```

Comment:
We don't need a feature flag if I refactor all of the examples in this branch, keep in mind that this would make the diff a bit bigger though. But if you don't mind that and you think that it is not nice to add a feature flag only to remove it in the next branch, then I will refactor all of the examples to use the `DocsExample` component now.